aodh/server.sls

```diff
@@ -1,4 +1,25 @@
 {%- from "aodh/map.jinja" import server with context %}
 {%- if server.enabled %}
+# Exclude unsupported openstack versions
+{%- if server.version >= "mitaka"  %}
```

Comment:
I wouldn't even mind about testing this. If you try to apply the aodh roles to a liberty (or earlier) environment, it should crash. But maybe @cznewt has other views...